modules/modules.nix

```
@@ -36,6 +36,7 @@ let
     (loadModule ./misc/pam.nix { })
     (loadModule ./misc/qt.nix { })
     (loadModule ./misc/submodule-support.nix { })
+    (loadModule ./misc/tmpfiles.nix { })
```

You'll need to add
```suggestion
    (loadModule ./misc/tmpfiles.nix { condition = hostPlatform.isLinux; })
```
since systemd won't work on macos. See the CI error log.